### PR TITLE
Update for Mesa v1.0.0

### PR DESCRIPTION
**Pull request by wang-boyu**
Fixes https://github.com/projectmesa/mesa-geo/issues/77

### CODE REVIEW

File: examples/geo_schelling/server.py [hunk 1]
```diff
@@ -19,10 +19,8 @@ def render(self, model):
 
 
 model_params = {
-    "density": UserSettableParameter("slider", "Agent density", 0.6, 0.1, 1.0, 0.1),
-    "minority_pc": UserSettableParameter(
-        "slider", "Fraction minority", 0.2, 0.00, 1.0, 0.05
-    ),
+    "density": Slider("Agent density", 0.6, 0.1, 1.0, 0.1),
```
Comment (rht) [inline examples/geo_schelling/server.py:22]: This should have been `mesa.visualization.Slider`. We flatten every viz stuff into `mesa.visualization`.

Comment (wang-boyu) [reply examples/geo_schelling/server.py:22]: Thanks! Didn't know that.

Strangely I got `ImportError: cannot import name 'Slider' from 'mesa.visualization'` when doing `from mesa.visualization import Slider`, but `mesa.visualization.Slider` works. Is this expected?

Comment (rht) [reply examples/geo_schelling/server.py:22]: Yes, that is expected.

Comment (wang-boyu) [reply examples/geo_schelling/server.py]: Ok, this is interesting. I've changed it to `mesa.visualization.Slider`.

**Pull request merged**